main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```

Comment:
The game should loop so the user can create 10 stories. The while loop can work without round bracket.
```suggestion
while loop < 11:
```